src/renderer/browser/values/protocols.js

```diff
@@ -0,0 +1,11 @@
+export const CUSTOM_PROTOCOLS = {
+  NOS: {
+    tld: '.neo',
+    protocol: 'nos:'
+  }
+};
+
+export const PROTOCOLS = {
+  HTTPS: 'https:',
+  HTTP: 'http:'
+};
```

Comment:
I would just export these directly since we don't ever do anything with the object directly, i.e.:

```js
export const HTTPS = 'https:';
export const HTTP = 'http:';
```

Then you can just import via:

```js
import { HTTP, CUSTOM_PROTOCOLS } from '../values/protocols';
```